src/github/middleware.ts

```diff
@@ -235,12 +237,20 @@ export default (
 			try {
 				await callback(context, jiraClient, util);
 			} catch (err) {
-				context.log.error(
-					err,
-					`Error processing the event for Jira hostname '${jiraHost}'`
-				);
-				emitWebhookFailedMetrics(webhookEvent);
-				context.sentry?.captureException(err);
+				const isWarning = warnOnErrorCodes.find(code => err.message.includes(code));
+				if(!isWarning) {
+					context.log.error(
+						err,
+						`Error processing the event for Jira hostname '${jiraHost}'`
+					);
+					emitWebhookFailedMetrics(webhookEvent);
+					context.sentry?.captureException(err);
+				} else {
+					context.log.info(
```

Comment:
If this is a warning we should probably change `.info` to `.warn`

- [ ] change `.info` to `.warn`